packages/server/src/oauth/utils.ts

```diff
@@ -703,6 +704,11 @@ export async function getExternalUserInfo(
     throw new OperationOutcomeError(badRequest('Failed to verify code - check your identity provider configuration'));
   }
 
+  if (response.status === 429) {
+    logger.warn('Too many requests sent in a given amount of time', response.status);
```

Comment:
This log message is more verbose than necessary; you could consider making it shorter and more information-dense like e.g. "Auth rate limit exceeded".  Also, adding `response.status` to the log probably isn't very useful, since it's guaranteed to always be `429` here